src/atoms/Lines/lines.tsx

```diff
@@ -0,0 +1,7 @@
+import { Styled } from '../../theme';
+
+export const Line = Styled.div<{ vertical?: boolean, length?: string }>`
```

Comment:
I see this in a lot of places in the app, lets have it consistent in case color needs to change or smth.